octomap_planner_client:
  plan_and_fly: false